intellij-plugin-structure/structure-dotnet/src/main/kotlin/com/jetbrains/plugin/structure/dotnet/ReSharperPluginManager.kt

```diff
@@ -116,6 +116,7 @@ class ReSharperPluginManager private constructor(private val extractDirectory: P
       idParts.size > 1 -> idParts[1]
       else -> id
     }
+    val nuspecFileName = "$id.nuspec"
```

Comment:
Shouldn't we get the real name here also (from `descriptorFile.fileName`)?

Reply:
Well, we will rename it in structure to "$id.nuspec", so...